src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue

```diff
@@ -533,20 +542,20 @@ onMounted(() => {
                         <div class="row">
                           <div class="col-xs-4 col-xl-8 endpoint-name">
                             <div class="row box-header">
-                              <div class="col-lg-max-9 no-side-padding lead righ-side-ellipsis" :title="instance.name">
+                              <div class="col-lg-max-9 no-side-padding lead righ-side-ellipsis" v-tooltip :title="instance.name">
                                 {{ instance.name }}
                               </div>
                               <div class="col-lg-4 no-side-padding endpoint-status">
                                 <span class="warning" v-if="formatGraphDuration(instance.metrics.criticalTime).value < 0">
-                                  <i class="fa pa-warning" :title="`Warning: instance currently has negative critical time, possibly because of a clock drift.`"></i>
+                                  <i class="fa pa-warning" v-tooltip :title="`Warning: instance currently has negative critical time, possibly because of a clock drift.`"></i>
                                 </span>
                                 <span class="warning" v-if="instance.isScMonitoringDisconnected">
-                                  <i class="fa pa-monitoring-lost endpoint-details" :title="`Unable to connect to monitoring server`"></i>
+                                  <i class="fa pa-monitoring-lost endpoint-details" v-tooltip :title="`Unable to connect to monitoring server`"></i>
                                 </span>
                                 <span class="warning" v-if="instance.isStale">
-                                  <i class="fa pa-endpoint-lost endpoint-details" :title="`Unable to connect to instance`"></i>
+                                  <i class="fa pa-endpoint-lost endpoint-details" v-tooltip :title="`Unable to connect to instance`"></i>
                                 </span>
-                                <span class="warning" v-if="instance.errorCount" :title="`${instance.errorCount} failed messages associated with this endpoint. Click to see list.`">
+                                <span class="warning" v-if="instance.errorCount" v-tooltip :title="instance.errorCount+` failed messages associated with this endpoint. Click to see list.`">
```

Comment:
Why the plus sign is needed here?

Reply:
I am concatenating the value of "instance.erroCount " and a string text.